src/Spectre.libClassifier.Tests/OpenCvDatasetTest.cpp

```diff
@@ -49,9 +52,9 @@ TEST_F(OpenCvDatasetInitializationTest, correct_dataset_opencv_initialization)
     EXPECT_NO_THROW(OpenCvDataset(data_long, labels));
 }
 
-TEST_F(OpenCvDatasetInitializationTest, throws_for_inconsistent_size)
+TEST_F(OpenCvDatasetInitializationTest, throws_for_empty_data)
```

Comment:
What about `InconsistentArgumentSizesException`? It was tested, but now it's not.

Reply:
done